src/main/java/seedu/address/logic/parser/MatchMateParser.java

```diff
@@ -105,8 +113,8 @@ public Command parseCommand(String userInput) throws ParseException {
      * Checks for prefixes if they are valid with the currently supported prefix codes. The argument is valid
      * if every prefix occurence in {@code argsString} is lead and trailed by a whitespace.
      *
-     * @param argsString      Arguments string of the form: {@code preamble <prefix> value <prefix> value ...}
-     * @return                Boolean object that indicates whether {@code argsString} is valid.
+     * @param argsString Arguments string of the form: {@code preamble <prefix> value <prefix> value ...}
+     * @return Boolean object that indicates whether {@code argsString} is valid.
```

Comment:
The return type is already in the code, should omit `Boolean object that indicates` since it's obvious.